.github/workflows/build_and_publish_dev.yaml

```
@@ -0,0 +1,28 @@
+name: Build image (Dev)
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  publish-source-stackadapt-image:
+    runs-on: ubuntu-latest
```

```suggestion
    if: ${{ contains(github.event.pull_request.labels.*.name,'dev_deploy') }}
```

Can we add this tidbit so the dev deploy only happens if the engineer applies the `dev-deploy` label? It's not super important but would be consistent with dev deploy behavior in other repos.